content/tutorials/grafana-fundamentals.md

```diff
@@ -59,13 +70,15 @@ docker-compose up -d
 
 The first time you run `docker-compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
 
+> **Note:** If you already have Grafana, Loki, or Prometheus running on your system, then you might see errors because the Docker image is trying to use ports that your local installations are already using. Stop the services, then run 
+
 - Ensure all services are up-and-running:
 
 ```
 docker-compose ps
 ```
 
-All services should report their state as "Up".
+All services should report their status as "Up <time>".
```

Comment:
All the services should say "Up" in the State column.

```
              Name                            Command               State               Ports             
----------------------------------------------------------------------------------------------------------
monitoring-workshop_app_1          /bin/sh -c /app http://db  ...   Up      0.0.0.0:8081->80/tcp          
monitoring-workshop_db_1           /db                              Up      0.0.0.0:8082->80/tcp          
monitoring-workshop_grafana_1      /run.sh                          Up      0.0.0.0:3001->3000/tcp        
monitoring-workshop_loki_1         /usr/bin/loki -config.file ...   Up      0.0.0.0:3100->3100/tcp, 80/tcp
monitoring-workshop_prometheus_1   /bin/prometheus --config.f ...   Up      0.0.0.0:9090->9090/tcp        
monitoring-workshop_promtail_1     /usr/bin/promtail -config. ...   Up                                    
```

Reply:
Mine displayed up and the amount of time up, and the column name was Status.

ARGH, inconsistent UI for the lose.